grpc/build.gradle

```diff
@@ -69,5 +89,13 @@ idea {
         sourceDirs += file("${protobuf.generatedFilesBaseDir}/main/grpc");
         sourceDirs += file("${protobuf.generatedFilesBaseDir}/test/java");
         sourceDirs += file("${protobuf.generatedFilesBaseDir}/test/grpc");
+        sourceDirs += file("${protobuf.generatedFilesBaseDir}/jmh/java");
+        sourceDirs += file("${protobuf.generatedFilesBaseDir}/jmh/grpc");
     }
 }
+
+tasks.checkstyleJmh.enabled = false
```

Comment:
Why were JMH classes disabled? Just curious.

Reply:
Didn't want to deal with javadoc, but realized I could edit the checkstyle config so they're treated as tests, fixed.